packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -385,4 +389,54 @@ export class UnfurlService {
 
         return imageUrl;
     }
+
+    async exportDashboard(
+        dashboardUuid: string,
+        user: SessionUser,
+    ): Promise<NotificationPayloadBase['page']> {
+        const dashboard = await this.dashboardModel.getById(dashboardUuid);
+        const {
+            url,
+            minimalUrl,
+            pageType,
+            details,
+            organizationUuid,
+            projectUuid,
+        } = {
+            url: `${this.lightdashConfig.siteUrl}/projects/${dashboard.projectUuid}/dashboards/${dashboardUuid}/view`,
+            minimalUrl: `${this.lightdashConfig.siteUrl}/minimal/projects/${dashboard.projectUuid}/dashboards/${dashboardUuid}`,
+            details: {
+                name: dashboard.name,
+                description: dashboard.description,
+            },
+            pageType: LightdashPage.DASHBOARD,
+            organizationUuid: dashboard.organizationUuid,
+            projectUuid: dashboard.projectUuid,
+        };
+        if (
+            user.ability.cannot(
+                'view',
+                subject('Dashboard', { organizationUuid, projectUuid }),
+            )
+        ) {
+            throw new ForbiddenError();
+        }
+        const imageUrl = await this.unfurlImage(
+            minimalUrl,
+            pageType,
+            `${details.name.toLowerCase().replace(' ', '-')}`,
```

Comment:
We might need a write a util function that converts a string into a valid filename. (removes special charts, removes white spaces and lowercase letters)
And this name needs to be unique(using nanoId), otherwise, there might be issues when exporting the same dashboard multiple times.